src/assets/drizzle/styles/components/nav.css

```diff
@@ -1,103 +1,84 @@
 :root {
-  --Nav-background: var(--color-white);
-  --Nav-border: 2px solid var(--color-silver);
-  --Nav-active-color: inherit;
-  --Nav-active-border: var(--color-blue);
+  --Nav-background: var(--color-silver);
+  --Nav-active-background: var(--color-white);
+  --Nav-header-background: var(--color-blue);
+  --Nav-header-color: var(--color-white);
+  --Nav-toggle-size: 3em;
+  --Nav-item-padding: calc(var(--space) * 0.75) var(--space);
+  --Nav-item-nested-indent: calc(var(--space) * 2);
+  --Nav-item-nested-font-size: 0.75em;
 }
 
 .Nav {
   background: var(--Nav-background);
-  border: var(--Nav-border);
-  border-width: 0 0 2px;
-
-  @media (--lg-viewport) {
-    border-width: 0 2px 0;
-  }
-}
-
-.Nav a,
-.Nav a:--enter {
-  text-decoration: none;
 }
 
 .Nav-header {
-  padding: 1rem;
+  background: var(--Nav-header-background);
+  color: var(--Nav-header-color);
+  padding: calc(var(--space) * 0.75) var(--space);
   position: relative;
 }
 
-.Nav-header-logo {
-  font-weight: bold;
-
-  & > img {
-    vertical-align: bottom;
-  }
-}
-
 .Nav-header-toggle {
-  background: transparent;
-  border: 0;
-  box-shadow: none;
-  cursor: pointer;
-  font: inherit;
-  margin: -1.5em 0 0;
-  padding: 1em;
   position: absolute;
-  top: 50%;
+  width: var(--Nav-toggle-size);
+  top: 0;
   right: 0;
+  bottom: 0;
 
   @media (--lg-viewport) {
     display: none;
   }
 
-  & svg {
-    display: block;
-    fill: currentColor;
-    height: 1em;
-    opacity: 0.5;
-    width: 1em;
+  & > svg {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    margin: auto;
   }
+}
 
-  @nest .Nav.is-active & {
-    transform: rotate(180deg);
+.Nav a,
+.Nav a:--enter {
```

Comment:
Had to look up what `--enter` was. LOL _Nice!_ (I think I can use this on _my_ PR. :grin:)